src/PKSim.Presentation/Presenters/Protocols/AdvancedProtocolPresenter.cs

```diff
@@ -30,28 +30,38 @@ public interface IAdvancedProtocolPresenter : IPresenter<IAdvancedProtocolView>,
       void RemoveSchema(SchemaDTO schemaToRemove);
       void SetApplicationType(SchemaItemDTO schemaItemDTO, ApplicationType newApplicationType);
       void SetFormulationType(SchemaItemDTO schemaItemDTO, string newFormulationType);
-      void AddSchemaItemTo(SchemaDTO schemaDTO, SchemaItemDTO schemaItemDTOToDupicate);
+      void AddSchemaItemTo(SchemaDTO schemaDTO, SchemaItemDTO schemaItemDTOToDuplicate);
       void RemoveSchemaItem(SchemaItemDTO schemaItemDTO);
-      IList DynamicParametersFor(SchemaItemDTO schemaItemDTO);
-      bool HasDynamicParameters(SchemaItemDTO schemaItemDTO);
+      IList DynamicContentFor(SchemaItemDTO schemaItemDTO);
```

Comment:
Dynamic content can be something else than parameters now. So I renamed it to be more generic